packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -130,8 +130,8 @@ void main() {
     final BoxDecoration collapsingContainerDecoration = getContainer(collapsedKey).decoration! as BoxDecoration;
     expect(collapsingContainerDecoration.color, Colors.transparent);
     // Opacity should change but color component should remain the same.
-    expect(collapsingContainerDecoration.border!.top.color, const Color(0x15333333));
-    expect(collapsingContainerDecoration.border!.bottom.color, const Color(0x15333333));
+    expect(collapsingContainerDecoration.border!.top.color, const Color(0x15222222));
```

Comment:
Why does this test change?

Reply:
well when i executed the test file there is an assert error with color of the border. I thought the animation tween of border and color are different. so, i updated the color value

Reply:
But why does the color change? The comment there says it shouldn't. This test should still behave the same way with your changes, correct?

Reply:
```
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
The following TestFailure was thrown running a test:
Expected: Color:<Color(0x15333333)>
  Actual: Color:<Color(0x15222222)>

When the exception was thrown, this was the stack:
#4      main.<anonymous closure> (file:///Users/padya/MySpace/flutter-project/flutter/packages/flutter/test/material/expansion_tile_test.dart:133:5)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)

This was caught by the test expectation on the following line:
  file:///Users/padya/MySpace/flutter-project/flutter/packages/flutter/test/material/expansion_tile_test.dart line 133
The test description was:
  ExpansionTile initial state (variant: TargetPlatform.macOS)
════════════════════════════════════════════════════════════════════════════════════════════════════
```

Reply:
Color(0x00000000)
Color(0x04060606)
Color(0x080d0d0d)
Color(0x0b131313)
Color(0x0f181818)
Color(0x121e1e1e)
Color(0x15222222)
Color(0x17272727)
Color(0x1a2b2b2b)
Color(0x1c2e2e2e)
Color(0x1d303030)
Color(0x1e323232)
Color(0x1e323232)
Color(0x1f333333)



This is the. color changing pattern for the dividec color `Color(0x1f333333)`

Reply:
idk why the color is changing though

Reply:
Previously there is no start color for the border